packages/transport/src/modules/sync/externalEventProcessors/RelationshipReactivationCompletedExternalEventProcessor.ts

```diff
@@ -0,0 +1,21 @@
+import { Serializable, serialize, validate } from "@js-soft/ts-serval";
+import { RelationshipChangedEvent } from "../../../events";
+import { Relationship } from "../../relationships/local/Relationship";
+import { ExternalEvent } from "../data/ExternalEvent";
+import { ExternalEventProcessor } from "./ExternalEventProcessor";
+
+class RelationshipReactivationCompletedExternalEventData extends Serializable {
+    @serialize()
+    @validate()
+    public relationshipId: string;
+}
+
+export class RelationshipReactivationCompletedExternalEventProcessor extends ExternalEventProcessor {
+    public override async execute(externalEvent: ExternalEvent): Promise<Relationship | undefined> {
+        const payload = RelationshipReactivationCompletedExternalEventData.fromAny(externalEvent.payload);
+        const relationship = await this.accountController.relationships.applyRelationshipChangedEvent(payload.relationshipId);
+
+        this.eventBus.publish(new RelationshipChangedEvent(this.ownAddress, relationship));
```

Comment:
I think it would be great to have an additional event to be published on the eventbus (RelationshipReactivationCompleted)

Reply:
please make sure to also add the new event in the runtime event proxy